14_functions_lab/01.py

```diff
@@ -1,7 +1,21 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+"""
+"""
+
+def mysum(*numbers):
+    sum = 0
+    for number in numbers:
+        if type(number) is int: 

```

Comment:
A better way to check if something is a number is:

```
isinstance(n, numbers.Number)
```

This also works for floats